plugins/grpc/cpp/v1.51.1/buf.plugin.yaml

```diff
@@ -3,6 +3,8 @@ name: buf.build/grpc/cpp
 plugin_version: v1.51.1
 source_url: https://github.com/grpc/grpc
 description: Generates C++ client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v21.12
```

Comment:
I didn't backfill these changes back through all the older versions of these plugins - just the latest version.